Scripts/developer_scripts/add_toc_to_github_wiki_page.py

```diff
@@ -1,125 +1,130 @@
-from sys import argv
-from sys import exit
 import codecs
 import re
 import argparse
+import sys
+from urllib.parse import quote
 
 parser = argparse.ArgumentParser()
-parser.add_argument("filename",
-                    help="the Mardown file to process")
-parser.add_argument("--codebase",
-                    help="for a Markdown file of Codebase instead of Github",
-                    action="store_true")
-parser.add_argument("--h1",
-                    help="support level one sections (h1)",
-                    action="store_true")
+parser.add_argument("filename", help="the Mardown file to process")
+parser.add_argument(
+    "--codebase",
+    help="for a Markdown file of Codebase instead of Github",
+    action="store_true",
```

Comment:
?

Reply:
That option was already in `master` before this PR. It allows to deal with Codebase Notebooks. The diff is here because of the reformatting using `black`.

Reply:
Use the option to see the diff without the whitespace changes: https://github.com/CGAL/cgal/pull/7867/files?w=1

Reply:
for sure that extra useless comma was not there...

Reply:
It was probably added by the Python formatter `black`.

Reply:
That is a feature of the Blake code formatter. It seems to be an idiom in Python:  [PEP 8 When to Use Trailing Commas](https://peps.python.org/pep-0008/#when-to-use-trailing-commas).